tests/state/test_cache.py

```diff
@@ -213,7 +213,7 @@ def worker(timeout: int) -> bytes:
     with pytest.raises(TimeoutError):
         waiter_fast.result()
 
-    with pytest.raises(exc):
+    with pytest.raises((ExecutionTimeoutError, TigerExecutionTimeoutError)):
```

Comment:
Since you are testing against one of several fixed exceptions then you can remove the parametrized exc being passed in since its not used anymore.

Reply:
or move the tuple into the parameterization